### DISCUSSION TITLE
Mi pregunta

### DISCUSSION THREAD

**Discussion by AldrickSalazar** [Launch-X-Latam/MisionFrontEnd#21] [Q&A]
¿Alguien ya sabe sobre estos temas?

**Comment by Emilio17MC** [comment]
Yo sobre algunos, pero todos estamos aquí para aprender!

**Comment by JosafatJimenezB** [comment]
si, lo que necesites compañero, aqui aprendemos todos

**Comment by IngDavidHoyosGil** [comment]
Yo solo sé cosas muy basicas de Javascript, pero aqui estamos todos para aprender

**Comment by OscarGUD** [comment]
SI un poco, cualquier apoyo aqui estoy .😉

**Comment by samujm** [comment]
No todo, pero se aprende :)

**Comment by gersonjeanpierre** [comment]
Pues HTML, CSS, Javascript con Typescript y ahi siguiendo, practicando y aprendiendo porque yo solo se que nada se jeje

**Comment by JessTeran** [comment]
Nou, Pero aquí estoy tratando de aprender

**Comment by rromi182** [comment]
Algunas cosas básicas y queriendo aprender más siempre

**Comment by DaviiidSG** [comment]
Únicamente HTML, CSS y lo básico de JS pero aquí todos vamos a aprender y entre todos nos vamos a apoyar :))

**Comment by LaMazorca** [comment]
Estoy igual solo HTMl, CSS y Javascript muy básico, todo de manera autodidacta, pero se hace el esfuerzo

**Comment by Allegseu95** [comment]
Yo si, la gran mayoria pero aquí estamos para aprender y reforzar conocimientos, asi que para cualquier ayuda que se les pueda brindar no duden en preguntar.

**Comment by OATS74** [comment]
no consegui ver la sección de hoy y si solo se que no sé nada .jjj

**Comment by OATS74** [comment]
Por donde se conectaron para ver la clase de hoy . YouTube o por donde estoy en el aire

**Comment by doguedogue** [comment]
Pos un poco, pero tienes que profundizar por tu cuenta, aprovecha cada semana para aprender más sobre el tema que se trate, haz las prácticas y verás los resultados.

Vamos aqui estamos para las dudas (que no estén en google ni en stackoverflow)

**Comment by RocioSanchez2022** [comment]
Si, conozco algunos temas. Pero todos estamos acá para aprender!
